ziti/edge/types.go

```diff
@@ -92,6 +92,13 @@ type Service struct {
 	Tags           map[string]string                 `json:"tags"`
 }
 
+type Terminator struct {
+	Id        string `json:"id"`
+	ServiceId string `json:"serviceId"`
+	RouterId  string `json:"routerId"`
+	Identity  string `json:"Identity"`
```

Comment:
why does this return an identity and not just the id?

Reply:
maybe it's just misnamed? also is the casing difference historical? (capital I)

Reply:
This is that whole terminator identity vs edge identity thing. It's the terminator identity, which we should rename to something less confusing. But for now, it's still Identity